source/tools/Makefile

```
@@ -21,26 +15,24 @@ endef
 define assemble
 	@printf "\n\033[0;36mAssembling %s\033[0m\n\n" $(1)
-	@$(N80) $(1) $(2)
+	@$(N80) $(N80_ARGS) $(1) $(2) $(3)
 endef
 
 COMS := CONCLUS.COM DELALL.COM DEVINFO.COM DRIVERS.COM DRVINFO.COM FASTOUT.COM LOCK.COM MAPDRV.COM NSYSVER.COM RALLOC.COM
 INCS := ../kernel/macros.inc ../kernel/const.inc
 RELS := CODES.REL DATA.REL SHARED.REL
 
-asm-tools: $(COMS)
+asm-tools: prerequisites $(COMS)
```

checkmake (0.3.2)

[warning] Target "asm-tools" should be declared PHONY.

(phonydeclared)